src/components/sbb-dialog/readme.md

```diff
@@ -0,0 +1,131 @@
+# sbb-dialog
+The dialog component provides a way to present content on top of the app's content and can be used in several contexts. The dialog offers the following features:
+
+- creates a backdrop, for disabling interaction below the modal;
+- disables scrolling of the page content while open;
+- manages focus properly by setting it on the first focusable element;
+- can have a header and a footer, both of which are optional;
+- can host an `sbb-action-group` component in the footer;
+- has a close button, which is always visible;
+- can display a back button next to the title;
+- adds the appropriate ARIA roles automatically.
+
+The dialog box can be dismissed by clicking on the close button, clicking on the backdrop, or pressing the `Esc` key.
+
+## Usage
+In order to show a modal you need to call the `present(event?: PointerEvent)` method on the `sbb-dialog` component:
+
+```html
+<sbb-button label="Open dialog" click="openDialog(event, 'my-dialog')"></sbb-button>
+<sbb-dialog id="my-dialog" title-content="Title" title-back-button="true">
+  Dialog content.
+  <div slot="action-group">...</div>
+</sbb-dialog>
+
+<script>
+  const openDialog = (event, id) => {
+    const dialog = document.getElementById(id);
+    dialog.present(event);
+  };
+</script>
+```
+
+Note that it is necessary to pass the event object to the `present()` method to allow the dialog to detect whether it has been opened by click or keyboard, so that the focus can be better handled.
```

Comment:
```suggestion
Note that it is necessary to pass the event object to the `open()` method to allow the dialog to detect whether it has been opened by click or keyboard, so that the focus can be better handled.
```